README.md

```diff
@@ -14,21 +14,23 @@ See the [gnome extensions page](https://extensions.gnome.org/extension/1500/cont
 
 # Install from source
 
-First pick the right branch by the gnome-shell-X.XX version: 
+First pick the right branch by the `gnome-shell-X.XX` version: 
 
 | branch | gnome-shell version |
 | --- | --- |
 | master | 3.36.1 |
+| 3.36   | 3.36.4 |
```

Comment:
The podman version is something that needs handling as well. I'm thinking on adding per podman release json parsing handling. Some people may newer version of gnome but old podman versions and vice versa.